recipes/oatpp-sqlite/all/test_package/CMakeLists.txt

```
@@ -1,12 +1,12 @@
 cmake_minimum_required(VERSION 3.1)
-project(oatpp-sqlite-test CXX)
+project(test_package CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
 find_package(oatpp-sqlite REQUIRED)
 
-add_executable(oatpp-sqlite-test oatpp-sqlite-test.cpp)
+add_executable(${PROJECT_NAME} test_package.cpp)
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
 
-target_link_libraries(oatpp-sqlite-test PRIVATE oatpp::oatpp-sqlite)
+target_link_libraries(${PROJECT_NAME} PRIVATE CONAN_PKG::oatpp-sqlite)
```

Same as https://github.com/conan-io/conan-center-index/pull/3579/files#r524642941

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE oatpp::oatpp-sqlite)
```